evalml/pipelines/utils.py

```diff
@@ -74,6 +75,16 @@ def _get_drop_all_null(X, y, problem_type, estimator_class, sampler_name=None):
     return component
 
 
+def _get_replace_null(X, y, problem_type, estimator_class, sampler_name=None):
```

Comment:
I think we need to modify `_get_ohe` to add a ohe not encoder when there are boolean nullable present. A good test case would be a dataset with only Double and BooleanNullable features.